rfd/0109-cloud-agent-upgrades.md

```diff
@@ -327,6 +327,33 @@ to be changed to follow the same pattern of registering a release channel and in
 by ec2 discovery.
 
 
+### ECS Fargate Agents
```

Comment:
I think we should rename the section to refer the integration subkind: AWS OIDC; and the associated action: Deploy Service.
So, maybe something like `AWS OIDC - Deploy Service Agents (Amazon ECS with Fargate)`